.github/workflows/e2e.yml

```diff
@@ -38,6 +38,35 @@ jobs:
       || (contains(github.event.pull_request.labels.*.name, 'build')
           && (github.event.action != 'labeled' || github.event.label.name == 'build')
           )
+    steps:
+      - name: Get Secrets from GCP Secret Manager
+        id: 'secrets'
+        uses: 'google-github-actions/get-secretmanager-secrets@v0'
+        with:
+          secrets: |-
```

Comment:
It would be good to fetch only those secrets depending on the module rather than all.